baseline/PLS_SM/full_flow.py

```diff
@@ -376,7 +376,7 @@ def test(
             # save
             pred_df = pd.DataFrame(pred, index=Y.index)
 
-            # Check for NaNs in Y[oxide]
+            # Check that the dataframe has at least one row and that all major oxides are columns
```

Comment:
```suggestion
            # Check for NaNs in Y[oxide]
```
I think this was updated wrongly by mistake